content/docs/handling-events.md

```diff
@@ -56,7 +56,7 @@ function ActionLink() {
 
 Here, `e` is a synthetic event. React defines these synthetic events according to the [W3C spec](https://www.w3.org/TR/DOM-Level-3-Events/), so you don't need to worry about cross-browser compatibility. See the [`SyntheticEvent`](/docs/events.html) reference guide to learn more.
 
-When using React you should generally not need to call `addEventListener` to add listeners to a DOM element after it is created. Instead, just provide a listener when the element is initially rendered.
+When using React, you should generally don't need to call `addEventListener` to add listeners to a DOM element after it is created. Instead, just provide a listener when the element is initially rendered.
```

Comment:
Typo: either use "should" or "don't", not both

The added commas are good to go

Reply:
Fixed it.